### ISSUE TITLE
The Help Page in the Site does not navigate

### ISSUE THREAD

**Issue by gregoryagu** [aurelia-ui-toolkits/aurelia-syncfusion-bridge#37]
If you go to the Help page, and then click on any link on the left, the app does not show the page.  It just stays on the intro page.

It seemed to work a few days ago when I was reading it.

I tested in both IE and Chrome with the same result.

**Comment by adriatic** [comment]
Is this a problem with https://aurelia-ui-toolkits.gitbooks.io/materialize-bridge-docs/content/about_this_application/introduction.html - since it is reported as an issue in Materialize repository? There is no Help page in this application?

**Comment by Thanood** [comment]
@adriatic http://aurelia-ui-toolkits.github.io/demo-syncfusion/#/help
Navigation does not work, indeed. No errors in dev console, except this before navigation:

```
doc.js:94 Uncaught TypeError: Cannot read property 'anchor' of undefined
```

It happens here:
https://github.com/aurelia-ui-toolkits/aurelia-syncfusion-bridge/blob/master/sample/src/help/doc.js#L94

`this.anchor` is not defined.

> since it is reported as an issue in Materialize repository?

Where is that? :smile:

**Comment by karthickthangasamy** [comment]
@Thanood routeChange subscription is not triggered from [here](https://github.com/aurelia-ui-toolkits/aurelia-syncfusion-bridge/blob/master/sample/src/help/doc.js#L16), but the same is working in bridge source. If the `routeChanged` method triggered, `this.params` won't be `undefined`.

**Comment by adriatic** [comment]
@Thanood - I was mistakenly assuming (have no idea why) that @gregoryagu reported a problem about Materialize bridge.

**Comment by karthickthangasamy** [comment]
This issue fixed and tested in Windows 10 Chrome v56.0.2924.87, Edge and IE 11 browsers.
